examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/AppClientTest.kt

```diff
@@ -51,15 +52,52 @@ class AppClientTest: RealmTest() {
     }
 
     @Test
-    fun changeBaseUrl() {
-        val defaultBaseUrl = "https://realm.mongodb.com"
-        val newBaseUrl = "https://example.org/"
-        val config = AppConfiguration.create(YOUR_APP_ID)
-        assertEquals(config.baseUrl, defaultBaseUrl)
-        val configWithNewBaseUrl = AppConfiguration.Builder(YOUR_APP_ID)
-            .baseUrl(newBaseUrl)
+    fun customBaseUrl() {
+        val defaultBaseUrl = "https://services.cloud.mongodb.com"
+        val newBaseUrl = "https://example.com"
+        val defaultConfig = AppConfiguration.create(YOUR_APP_ID)
+        assertEquals(defaultConfig.baseUrl, defaultBaseUrl)
+        // :snippet-start: custom-base-url
+        // Specify a baseUrl to connect to instead of the default
+        val config = AppConfiguration.Builder(YOUR_APP_ID)
+            .baseUrl("https://example.com")
             .build()
-        assertEquals(configWithNewBaseUrl.baseUrl, newBaseUrl)
+        // :snippet-end:
+        assertEquals(config.baseUrl, newBaseUrl)
+    }
+
+    // :snippet-start: experimental-opt-in
+    // Opt in to the experimental Edge Server API
+    @OptIn(ExperimentalEdgeServerApi::class)
+    // :snippet-end:
+    @Ignore
+    // TODO: Update when we get Edge Server running in a CI and can write automated tests for full flow
+    // Ignored until we can test in CI (was tested locally and succeeded)
+    fun changeBaseUrl() {
+        runBlocking {
+            val defaultBaseUrl = "https://services.cloud.mongodb.com"
+            val customBaseUrl = "http://localhost:80"
+            // :snippet-start: change-base-url
+            // Specify a custom baseUrl to connect to.
+            // In this case, an Edge Server instance running on the device.
+            val config = AppConfiguration.Builder(EDGE_SERVER_APP_ID)
+                .baseUrl("http://localhost:80")
+                .build()
+            val app = App.create(config)
+
+            // ... log in a user and use the app ...
+            // :remove-start:
+            assertEquals(app.baseUrl, customBaseUrl)
+            app.login(Credentials.anonymous())
+            assertNotNull(app.currentUser)
+            // :remove-end:
+
+            // Later, change the baseUrl to the default:
```

Comment:
In the copy for this example, you said:

> You can pass `null` to reset the `baseUrl` to the default value.

Might we want to use the same language in the comment here?

```suggestion
            // Later, pass `null` to reset the baseUrl to the default:
```